extensions/picocli/deployment/src/main/java/io/quarkus/picocli/deployment/PicocliProcessor.java

```diff
@@ -0,0 +1,96 @@
+package io.quarkus.picocli.deployment;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.jboss.jandex.AnnotationTarget;
+import org.jboss.jandex.ClassInfo;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.IndexView;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.arc.deployment.AnnotationsTransformerBuildItem;
+import io.quarkus.arc.deployment.BeanDefiningAnnotationBuildItem;
+import io.quarkus.arc.processor.AnnotationsTransformer;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.ApplicationIndexBuildItem;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.IndexDependencyBuildItem;
+import io.quarkus.deployment.builditem.QuarkusApplicationClassBuildItem;
+import io.quarkus.picocli.runtime.PicocliRunner;
+import io.quarkus.picocli.runtime.annotations.TopCommand;
+import io.quarkus.runtime.annotations.QuarkusMain;
+import picocli.CommandLine;
+
+class PicocliProcessor {
+
+    private static final String FEATURE = "picocli";
```

Comment:
Please move that one to the FeatureBuildItem with the other. They are sorted in alphabetical order.